Changes to doc (#897)
open-cluster-management/rhacm-docs: console/cluster_label.adoc

. From the navigation menu, click *Automate infrastructure* > *Clusters*.
. Find your cluster in the _Clusters_ table.

> . Click the *Options* icon to see more.

Reviewer: Another suggestion, when we instruct users to click a button, the name of the button is bold. Can you also ensure that the are 4 asterisks enclosing the name of the button

Reviewer: As stated via, Slack. 4 asterisks are not required to bold a term. 2 are sufficient. My mind was stuck in Markdown for a moment :P

. From the navigation menu, click *Automate infrastructure* > *Clusters*.
. Find your cluster in the _Clusters_ table.

> . Click the *overflow menu icon* (three dots on the right edge of the row) to see more.

Reviewer: Does `overflow menu` appear as the user hovers over the icon? I suggested confirming the name of the button

Author: <img width="313" alt="Screen Shot 2020-07-14 at 4 35 07 PM" src="https://user-images.githubusercontent.com/21374229/87473819-139b7700-c5f0-11ea-9bbf-e634f37fcf68.png">

Given this on hover, should we leave it as *Options*?

Reviewer: Yes, let's maintain `Options` icon

. Click *Edit Labels*.

> . From the _Edit Labels_ dialog box, enter a label in the form `Key=Value`. 

Reviewer: Update the statement to the following: . From the _Edit Labels_ dialog box, enter a label. Your entry might resemble the following content: Key=Value`.

Author: Should I remove the text that is directly below it: ". When done correctly, your label might resemble the following:"     environment=Dev"?

Reviewer: Yes, remove "When done correctly. I suggest replacing the `Key=Value` example with `environment=Dev`

. When done correctly, your label might resemble the following:

  ----
    environment=Dev
  ----

. Hit enter or space to additional labels.
. Click *Add*, then click *Done*.

* If you want to remove an existing label, select a label and click the *Remove* icon.
* If you want to update an existing label, you can reassign it's key to a new value using the method above.

> *Note* You can also edit a cluster label from a cluster's _Overview_ page. From the navigation menu, click *Automate infrastructure* > *Clusters*. 

Reviewer: If you are attempting to bold the Note here, you need 4 asterisks to enclose Note. I also suggest adding `:` after Note. For Example, **Note:**

. Find your cluster in the _Clusters_ table.
. Click the *Options* icon to see more.

> . Click *Edit Labels*.

Reviewer 2: The L is lowercase in both the menu action and the title of the dialog.

. From the _Edit Labels_ dialog box, enter a label. Your entry might resemble the following content: Key=Value`. 
. Hit enter or space to add additional labels.
. Click *Add*, then click *Done*.

> * If you want to remove an existing label, select a label and click the *Remove* icon.

Reviewer 2: "select a label" makes it sound like I need to click it first or something.
```suggestion
* If you want to remove an existing label, click the *Remove* icon next to the label in the list.
```

Reviewer 2: Should we also mention the undo feature here?

> * If you want to update an existing label, you can reassign it's key to a new value using the method above.

Reviewer 2: It's not clear what "the method above" is referring to, and it should be "its" not "it's"

I think we should spell out that you would add the value as if adding a new label, and it will overwrite the previous value.

----
  environment=Dev
----
* If you want to remove an existing label, select a label and click the *Remove* icon.
* If you want to update an existing label, you can reassign it's key to a new value using the method above.

. Click *Save*.

* If you want to remove an existing label, select a label and click the *Remove an item* icon.
* If you want to update an existing label, select and edit the label.
After editing your label, click the *Add a new item* icon.
+
Your label is updated.
+
Your created labels appear in the _Labels_ column.

> *Note* You can also edit a cluster label from the cluster _Overview_ page by clicking *Actions* > *Edit labels* or the _Label edit_ button.

Reviewer 2: The updates are live, right? I think we should mention that here. Something like "Changes to labels are saved automatically. Click **Done** to close the **Edit labels** dialog when you are finished."

> *Note*: You can also edit a cluster label from a cluster's _Overview_ page. From the navigation menu, click *Automate infrastructure* > *Clusters*. 

Reviewer 2: "Overview" should not be italicized or capitalized, as it is not an actual title. I think in most cases we refer to this as the "cluster details page".
